README.md

```diff
@@ -43,47 +33,25 @@ For Example:
 <md:BasicMetadata xmlns:md="http://www.movielabs.com/schema/md/v2.7/md" ContentID="urn:eidr:10.5240:7791-8534-2C23-9030-8610-5">
     <md:LocalizedInfo default="true" language="EN">
         <md:TitleDisplay60>The Name of a Movie</md:TitleDisplay60>
-        <md:TitleSort>Name of a Movie, The</md:TitleSort>
         <md:ArtReference resolution="224x320">https://via.placeholder.com/224x320</md:ArtReference>
         <md:ArtReference resolution="800x1200">https://via.placeholder.com/800x1200</md:ArtReference>
-        <md:Summary190>Summary of asset up to 190 characters in length.</md:Summary190>
-        <md:Summary400>Summary of asset up to 400 characters in length.</md:Summary400>
-        <md:CopyrightLine>castLabs 2019</md:CopyrightLine>
     </md:LocalizedInfo>
     <md:LocalizedInfo language="ES">
         <md:TitleDisplay60>El Nombre de una Pelicula</md:TitleDisplay60>
-        <md:TitleSort>Nombre de una Pelicula, El</md:TitleSort>
         <md:ArtReference resolution="224x320">https://via.placeholder.com/224x320</md:ArtReference>
         <md:ArtReference resolution="800x1200">https://via.placeholder.com/800x1200</md:ArtReference>
-        <md:Summary190>Resumen del activo hasta 190 caracteres de longitud.</md:Summary190>
-        <md:Summary400>Resumen del activo hasta 400 caracteres de longitud.</md:Summary400>
-        <md:CopyrightLine>castLabs 2019</md:CopyrightLine>
     </md:LocalizedInfo>
-    <md:RunLength>PT01M00.000S</md:RunLength>
     <md:ReleaseYear>2019</md:ReleaseYear>
     <md:WorkType>MOVIE</md:WorkType>
-    <md:People>
-        <md:Job>
-            <md:JobFunction>Actor</md:JobFunction>
-            <md:Character>Man on street #1</md:Character>
-        </md:Job>
-        <md:Name>Bruce Wayne</md:Name>
-    </md:People>
-    <md:People>
-        <md:Job>
-            <md:JobFunction>Actor</md:JobFunction>
-            <md:Character>Man on street #2</md:Character>
-        </md:Job>
-        <md:Name>Peter Parker</md:Name>
-    </md:People>
 </md:BasicMetadata>
 ```
 
-## Bonus
+## Bonus points
 
-Include an emulator mode, to emulate the look and feel of the asset metadata as it may be represented in a retailers digital storefront.
+1. Include an emulator mode, to emulate the look and feel of the asset metadata as it may be represented in a retailers digital storefront, e.g.: [Reference](https://www.flixstervideo.com/movies/pride-prejudice/urn:dece:cid:eidr-s:777C-E3FE-6FE0-83DA-315F-R)
+2. Implement backend for CRUD operations
```

Comment:
Plz make clear that this is optional but gives bonus points

Reply:
Are you referring only to CRUD or all bonus points in general?

Reply:
Now that you mention it that they are all bonus points... Nvmd!